injitest/src/main/java/inji/pages/AboutInjiPage.java

```diff
@@ -34,6 +43,17 @@ public boolean isAboutInjiHeaderDisplayed() {
         return this.isElementDisplayed(aboutInjiHeader);
     }
 
+    public boolean isAppIdVisible() {
+        String appId = getTextFromLocator(appID);
+        return appId.length() == 15;
+        }
+
+    public boolean isTuvaliVesionVisible() {
+        String appId = getTextFromLocator(tuvaliVesion);
+        return appId.length() == 21;
```

Comment:
are we checking AppId or tuvali version here. I see its tuvali version. Plz rename the variables accordingly.